## INDIAN START-UP ECOSYSTEM FUNDING ANALYSIS  (2018-2021)

This is a project to under study Start-Up businesses in the Indian Ecosystem to better understand and make data driven suggestions to my entity as to whether or not it is a ripe spacce to invest in. It is a comprehensive study of data sets spanning four different years (2018 - 2021) with information gathered on What business, Who funds them, Where they are located among many others. All these insights are supposed to help us drive us to a point where we make a data informed decision on whether to invest or not, based off the informative suggestion i provide to my team as a Data Analyst.

# BUSINESS UNDERSTANDING 

Summary of Task:

This project involves analyzing the funding received by start-ups in India from 2018 to 2021. The goal is to investigate the Indian start-up ecosystem and propose strategic recommendations based on data-driven insights. The datasets are provided for each year, and the analysis will cover start-up details, funding amounts, and investors' information. Data is stored across various sources, and it is crucial to gather, clean, and analyze this data to derive meaningful insights.

Libraries and Packages:

pandas for data manipulation and analysis
numpy for numerical operations
pyodbc for database connectivity
sqlalchemy for database ORM (optional)
matplotlib and seaborn for data visualization
scikit-learn for machine learning (if applicable)
python-dotenv for managing environment variables
requests for handling HTTP requests (if needed)
os and pathlib for handling file paths and directories


# BUSINESS QUESTIONS

To be able to come to the goal of our project as has been stated above, we will need to ask some questions, business-wise that is going to aid point us in the right direction with our goal. The following are the few that I have posed to help me reach the goal of my project. 

1.What sectors have shown the highest growth in terms of funding received over the past four years?

2.What geographical regions within India have emerged as the primary hubs for startup activity and investment, and what factors contribute to their prominence?

3.Are there any notable differences in funding patterns between early-stage startups and more established companies?

4.Which sectors recieve the lowest level of funding and which sectors recieve the highest levels of funding in India and what factors contribute to this?

5.Which investors have more impact on startups over the years?

6.What are the key characteristics of startups that successfully secure funding, and how do they differ from those that struggle to attract investment?

# HYPOTHESIS 

Our project is going to be to test two possible outcomes, of which one is the case with regards to the funding given to these start-ups. 

Null Hypothesis, (Ho) : There is no significant difference in the amount of funding between startups in particular "location".

Alternative Hypothesis, (Ha): There is a significant difference in the amount of funding between startups in "Blocation".

In [1]:
%pip install pyodbc
%pip install python-dotenv

Note: you may need to restart the kernel to use updated packages.
Note: you may need to restart the kernel to use updated packages.


In [2]:
import pandas as pd 
import numpy as np
import pyodbc 
import matplotlib.pyplot as plt 
from dotenv import dotenv_values 
import warnings 

warnings.filterwarnings('ignore')

# CONNECTING, COLLECTING AND LOADING DATASETS  

Importing Data Set into VSO Environment 

1. Databases for 2020 & 2021

In [3]:
# Load environment variables from .env file into a dictionary
environment_variables = dotenv_values('.env')
 
# Get the values for the credentials you set in the '.env' file
server = environment_variables.get("SERVER")
database = environment_variables.get("DATABASE")
username = environment_variables.get("UID")
password = environment_variables.get("PWD")

connection_string = f"DRIVER={{SQL Server}};SERVER={server};DATABASE={database};UID={username};PWD={password};MARS_Connection=yes;MinProtocolVersion=TLSv1.2;"

In [4]:
# Use the connect method of the pyodbc library and pass in the connection string.
# This will connect to the server and might take a few seconds to be complete.
 
connection = pyodbc.connect(connection_string)

In [5]:
# sql query to pull up data 
db_query ="""SELECT *
FROM INFORMATION_SCHEMA.TABLES
WHERE TABLE_TYPE = 'BASE TABLE' """

In [6]:
data = pd.read_sql(db_query,connection)

In [7]:
query = "Select * from dbo.LP1_startup_funding2020"
data20 = pd.read_sql(query, connection)
data20


Company_Brand  Founded HeadQuarter              Sector  \
0       Aqgromalin   2019.0     Chennai            AgriTech   
1         Krayonnz   2019.0   Bangalore              EdTech   
2     PadCare Labs   2018.0        Pune  Hygiene management   
3            NCOME   2020.0   New Delhi              Escrow   
4       Gramophone   2016.0      Indore            AgriTech   
...            ...      ...         ...                 ...   
1050  Leverage Edu      NaN       Delhi              Edtech   
1051         EpiFi      NaN        None             Fintech   
1052       Purplle   2012.0      Mumbai           Cosmetics   
1053        Shuttl   2015.0       Delhi           Transport   
1054         Pando   2017.0     Chennai            Logitech   

                                           What_it_does  \
0                          Cultivating Ideas for Profit   
1     An academy-guardian-scholar centric ecosystem ...   
2      Converting bio-hazardous waste to harmless waste   
3                          Escrow-as-a-service platform   
4     Gramophone is an AgTech platform enabling acce...   
...                                                 ...   
1050  AI enabled marketplace that provides career gu...   
1051  It offers customers with a single interface fo...   
1052         Online makeup and beauty products retailer   
1053                    App based bus aggregator serice   
1054            Networked logistics management software   

                                               Founders  \
0                       Prasanna Manogaran, Bharani C L   
1                      Saurabh Dixit, Gurudutt Upadhyay   
2                                       Ajinkya Dhariya   
3                                         Ritesh Tiwari   
4     Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   
...                                                 ...   
1050                                  Akshay Chaturvedi   
1051                    Sujith Narayanan, Sumit Gwalani   
1052                          Manish Taneja, Rahul Dash   
1053                      Amit Singh, Deepanshu Malviya   
1054                     Jayakrishnan, Abhijeet Manohar   

                                   Investor      Amount       Stage column10  
0                           Angel investors    200000.0        None     None  
1                           GSF Accelerator    100000.0    Pre-seed     None  
2                            Venture Center         NaN    Pre-seed     None  
3       Venture Catalysts, PointOne Capital    400000.0        None     None  
4       Siana Capital Management, Info Edge    340000.0        None     None  
...                                     ...         ...         ...      ...  
1050  DSG Consumer Partners, Blume Ventures   1500000.0        None     None  
1051          Sequoia India, Ribbit Capital  13200000.0  Seed Round     None  
1052                             Verlinvest   8000000.0        None     None  
1053             SIG Global India Fund LLP.   8043000.0    Series C     None  
1054                      Chiratae Ventures   9000000.0    Series A     None  

[1055 rows x 10 columns]

In [8]:
# pulling up the first few rows of the data 
data20.head()


Company_Brand  Founded HeadQuarter              Sector  \
0    Aqgromalin   2019.0     Chennai            AgriTech   
1      Krayonnz   2019.0   Bangalore              EdTech   
2  PadCare Labs   2018.0        Pune  Hygiene management   
3         NCOME   2020.0   New Delhi              Escrow   
4    Gramophone   2016.0      Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  
0                      Angel investors  200000.0      None     None  
1                      GSF Accelerator  100000.0  Pre-seed     None  
2                       Venture Center       NaN  Pre-seed     None  
3  Venture Catalysts, PointOne Capital  400000.0      None     None  
4  Siana Capital Management, Info Edge  340000.0      None     None

In [9]:
# Finding out necessary information about the dataframe 
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1055 non-null   object 
 1   Founded        842 non-null    float64
 2   HeadQuarter    961 non-null    object 
 3   Sector         1042 non-null   object 
 4   What_it_does   1055 non-null   object 
 5   Founders       1043 non-null   object 
 6   Investor       1017 non-null   object 
 7   Amount         801 non-null    float64
 8   Stage          591 non-null    object 
 9   column10       2 non-null      object 
dtypes: float64(2), object(8)
memory usage: 82.5+ KB


In [10]:
# finding out the structure of the dataframe 
# Number of rows and columns
data20.shape

(1055, 10)

In [11]:
# Creating a column to aid in identifying each data by the year they were added 
data20['Funding_Year'] = 2020

In [12]:
# changing the data type of the Funding year from foat64 to integer 
data20['Funding_Year'] = data20['Funding_Year'].astype(int)

In [13]:
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 11 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1055 non-null   object 
 1   Founded        842 non-null    float64
 2   HeadQuarter    961 non-null    object 
 3   Sector         1042 non-null   object 
 4   What_it_does   1055 non-null   object 
 5   Founders       1043 non-null   object 
 6   Investor       1017 non-null   object 
 7   Amount         801 non-null    float64
 8   Stage          591 non-null    object 
 9   column10       2 non-null      object 
 10  Funding_Year   1055 non-null   int32  
dtypes: float64(2), int32(1), object(8)
memory usage: 86.7+ KB


In [14]:
data20.head()

Company_Brand  Founded HeadQuarter              Sector  \
0    Aqgromalin   2019.0     Chennai            AgriTech   
1      Krayonnz   2019.0   Bangalore              EdTech   
2  PadCare Labs   2018.0        Pune  Hygiene management   
3         NCOME   2020.0   New Delhi              Escrow   
4    Gramophone   2016.0      Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  \
0                      Angel investors  200000.0      None     None   
1                      GSF Accelerator  100000.0  Pre-seed     None   
2                       Venture Center       NaN  Pre-seed     None   
3  Venture Catalysts, PointOne Capital  400000.0      None     None   
4  Siana Capital Management, Info Edge  340000.0      None     None   

   Funding_Year  
0          2020  
1          2020  
2          2020  
3          2020  
4          2020

In [15]:
# printing out dataframe column names 
print(data20.columns)

Index(['Company_Brand', 'Founded', 'HeadQuarter', 'Sector', 'What_it_does',
       'Founders', 'Investor', 'Amount', 'Stage', 'column10', 'Funding_Year'],
      dtype='object')


In [16]:

data = pd.read_sql(db_query,connection)

query1 = "Select * from dbo.LP1_startup_funding2021"
data21 = pd.read_sql(query1, connection)
data21


Company_Brand  Founded HeadQuarter                 Sector  \
0     Unbox Robotics   2019.0   Bangalore             AI startup   
1             upGrad   2015.0      Mumbai                 EdTech   
2        Lead School   2012.0      Mumbai                 EdTech   
3            Bizongo   2015.0      Mumbai         B2B E-commerce   
4           FypMoney   2021.0    Gurugram                FinTech   
...              ...      ...         ...                    ...   
1204        Gigforce   2019.0    Gurugram  Staffing & Recruiting   
1205          Vahdam   2015.0   New Delhi       Food & Beverages   
1206    Leap Finance   2019.0   Bangalore     Financial Services   
1207    CollegeDekho   2015.0    Gurugram                 EdTech   
1208          WeRize   2019.0   Bangalore     Financial Services   

                                           What_it_does  \
0     Unbox Robotics builds on-demand AI-driven ware...   
1        UpGrad is an online higher education platform.   
2     LEAD School offers technology based school tra...   
3     Bizongo is a business-to-business online marke...   
4     FypMoney is Digital NEO Bank for Teenagers, em...   
...                                                 ...   
1204                  A gig/on-demand staffing company.   
1205  VAHDAM is among the world’s first vertically i...   
1206  International education loans for high potenti...   
1207  Collegedekho.com is Student’s Partner, Friend ...   
1208  India’s first socially distributed full stack ...   

                                               Founders  \
0                           Pramod Ghadge, Shahid Memon   
1     Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                            Smita Deorah, Sumeet Mehta   
3               Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                         Kapil Banwari   
...                                                 ...   
1204                        Chirag Mittal, Anirudh Syal   
1205                                         Bala Sarda   
1206                         Arnav Kumar, Vaibhav Singh   
1207                                       Ruchir Arora   
1208                      Vishal Chopra, Himanshu Gupta   

                                           Investor        Amount  \
0                       BEENEXT, Entrepreneur First    $1,200,000   
1          Unilazer Ventures, IIFL Asset Management  $120,000,000   
2                  GSV Ventures, Westbridge Capital   $30,000,000   
3                            CDC Group, IDG Capital   $51,000,000   
4     Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000   
...                                             ...           ...   
1204                                Endiya Partners      $3000000   
1205                                       IIFL AMC     $20000000   
1206                                   Owl Ventures     $55000000   
1207               Winter Capital, ETS, Man Capital     $26000000   
1208                 3one4 Capital, Kalaari Capital      $8000000   

             Stage  
0     Pre-series A  
1             None  
2         Series D  
3         Series C  
4             Seed  
...            ...  
1204  Pre-series A  
1205      Series D  
1206      Series C  
1207      Series B  
1208      Series A  

[1209 rows x 9 columns]

In [17]:
data21.head()

Company_Brand  Founded HeadQuarter          Sector  \
0  Unbox Robotics   2019.0   Bangalore      AI startup   
1          upGrad   2015.0      Mumbai          EdTech   
2     Lead School   2012.0      Mumbai          EdTech   
3         Bizongo   2015.0      Mumbai  B2B E-commerce   
4        FypMoney   2021.0    Gurugram         FinTech   

                                        What_it_does  \
0  Unbox Robotics builds on-demand AI-driven ware...   
1     UpGrad is an online higher education platform.   
2  LEAD School offers technology based school tra...   
3  Bizongo is a business-to-business online marke...   
4  FypMoney is Digital NEO Bank for Teenagers, em...   

                                            Founders  \
0                        Pramod Ghadge, Shahid Memon   
1  Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                         Smita Deorah, Sumeet Mehta   
3            Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                      Kapil Banwari   

                                        Investor        Amount         Stage  
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A  
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None  
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D  
3                         CDC Group, IDG Capital   $51,000,000      Series C  
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed

In [18]:
data21.shape

(1209, 9)

In [19]:
data21.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1209 entries, 0 to 1208
Data columns (total 9 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1209 non-null   object 
 1   Founded        1208 non-null   float64
 2   HeadQuarter    1208 non-null   object 
 3   Sector         1209 non-null   object 
 4   What_it_does   1209 non-null   object 
 5   Founders       1205 non-null   object 
 6   Investor       1147 non-null   object 
 7   Amount         1206 non-null   object 
 8   Stage          781 non-null    object 
dtypes: float64(1), object(8)
memory usage: 85.1+ KB


In [20]:
# Creating a column to aid in identifying each data by the year they were added 
data21['Funding_Year'] = 2021

In [21]:
data21

Company_Brand  Founded HeadQuarter                 Sector  \
0     Unbox Robotics   2019.0   Bangalore             AI startup   
1             upGrad   2015.0      Mumbai                 EdTech   
2        Lead School   2012.0      Mumbai                 EdTech   
3            Bizongo   2015.0      Mumbai         B2B E-commerce   
4           FypMoney   2021.0    Gurugram                FinTech   
...              ...      ...         ...                    ...   
1204        Gigforce   2019.0    Gurugram  Staffing & Recruiting   
1205          Vahdam   2015.0   New Delhi       Food & Beverages   
1206    Leap Finance   2019.0   Bangalore     Financial Services   
1207    CollegeDekho   2015.0    Gurugram                 EdTech   
1208          WeRize   2019.0   Bangalore     Financial Services   

                                           What_it_does  \
0     Unbox Robotics builds on-demand AI-driven ware...   
1        UpGrad is an online higher education platform.   
2     LEAD School offers technology based school tra...   
3     Bizongo is a business-to-business online marke...   
4     FypMoney is Digital NEO Bank for Teenagers, em...   
...                                                 ...   
1204                  A gig/on-demand staffing company.   
1205  VAHDAM is among the world’s first vertically i...   
1206  International education loans for high potenti...   
1207  Collegedekho.com is Student’s Partner, Friend ...   
1208  India’s first socially distributed full stack ...   

                                               Founders  \
0                           Pramod Ghadge, Shahid Memon   
1     Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                            Smita Deorah, Sumeet Mehta   
3               Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                         Kapil Banwari   
...                                                 ...   
1204                        Chirag Mittal, Anirudh Syal   
1205                                         Bala Sarda   
1206                         Arnav Kumar, Vaibhav Singh   
1207                                       Ruchir Arora   
1208                      Vishal Chopra, Himanshu Gupta   

                                           Investor        Amount  \
0                       BEENEXT, Entrepreneur First    $1,200,000   
1          Unilazer Ventures, IIFL Asset Management  $120,000,000   
2                  GSV Ventures, Westbridge Capital   $30,000,000   
3                            CDC Group, IDG Capital   $51,000,000   
4     Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000   
...                                             ...           ...   
1204                                Endiya Partners      $3000000   
1205                                       IIFL AMC     $20000000   
1206                                   Owl Ventures     $55000000   
1207               Winter Capital, ETS, Man Capital     $26000000   
1208                 3one4 Capital, Kalaari Capital      $8000000   

             Stage  Funding_Year  
0     Pre-series A          2021  
1             None          2021  
2         Series D          2021  
3         Series C          2021  
4             Seed          2021  
...            ...           ...  
1204  Pre-series A          2021  
1205      Series D          2021  
1206      Series C          2021  
1207      Series B          2021  
1208      Series A          2021  

[1209 rows x 10 columns]

In [22]:
data21.head()

Company_Brand  Founded HeadQuarter          Sector  \
0  Unbox Robotics   2019.0   Bangalore      AI startup   
1          upGrad   2015.0      Mumbai          EdTech   
2     Lead School   2012.0      Mumbai          EdTech   
3         Bizongo   2015.0      Mumbai  B2B E-commerce   
4        FypMoney   2021.0    Gurugram         FinTech   

                                        What_it_does  \
0  Unbox Robotics builds on-demand AI-driven ware...   
1     UpGrad is an online higher education platform.   
2  LEAD School offers technology based school tra...   
3  Bizongo is a business-to-business online marke...   
4  FypMoney is Digital NEO Bank for Teenagers, em...   

                                            Founders  \
0                        Pramod Ghadge, Shahid Memon   
1  Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                         Smita Deorah, Sumeet Mehta   
3            Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                      Kapil Banwari   

                                        Investor        Amount         Stage  \
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A   
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None   
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D   
3                         CDC Group, IDG Capital   $51,000,000      Series C   
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed   

   Funding_Year  
0          2021  
1          2021  
2          2021  
3          2021  
4          2021

In [23]:
data21.shape

(1209, 10)

In [24]:
data21.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1209 entries, 0 to 1208
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1209 non-null   object 
 1   Founded        1208 non-null   float64
 2   HeadQuarter    1208 non-null   object 
 3   Sector         1209 non-null   object 
 4   What_it_does   1209 non-null   object 
 5   Founders       1205 non-null   object 
 6   Investor       1147 non-null   object 
 7   Amount         1206 non-null   object 
 8   Stage          781 non-null    object 
 9   Funding_Year   1209 non-null   int64  
dtypes: float64(1), int64(1), object(8)
memory usage: 94.6+ KB


In [25]:
# printing out dataframe column names 
print(data21.columns)

Index(['Company_Brand', 'Founded', 'HeadQuarter', 'Sector', 'What_it_does',
       'Founders', 'Investor', 'Amount', 'Stage', 'Funding_Year'],
      dtype='object')


Loading Data Sets Into VSO Environment 2 

2. Database Collection 2019

In [26]:
# Accessing database for 2019 from the One drive, now downloaded unto the local pc 
data19 = pd.read_csv('startup_funding2019.csv') 
data19

Company/Brand  Founded HeadQuarter                   Sector  \
0    Bombay Shaving      NaN         NaN                Ecommerce   
1         Ruangguru   2014.0      Mumbai                   Edtech   
2          Eduisfun      NaN      Mumbai                   Edtech   
3          HomeLane   2014.0     Chennai          Interior design   
4          Nu Genes   2004.0   Telangana                 AgriTech   
..              ...      ...         ...                      ...   
84     Infra.Market      NaN      Mumbai                Infratech   
85              Oyo   2013.0    Gurugram              Hospitality   
86       GoMechanic   2016.0       Delhi  Automobile & Technology   
87           Spinny   2015.0       Delhi               Automobile   
88  Ess Kay Fincorp      NaN   Rajasthan                  Banking   

                                         What it does  \
0          Provides a range of male grooming products   
1   A learning platform that provides topic-based ...   
2             It aims to make learning fun via games.   
3               Provides interior designing solutions   
4   It is a seed company engaged in production, pr...   
..                                                ...   
84  It connects client requirements to their suppl...   
85                Provides rooms for comfortable stay   
86  Find automobile repair and maintenance service...   
87                                Online car retailer   
88              Organised Non-Banking Finance Company   

                                             Founders  \
0                                  Shantanu Deshpande   
1               Adamas Belva Syah Devara, Iman Usman.   
2                                       Jatin Solanki   
3                        Srikanth Iyer, Rama Harinath   
4                              Narayana Reddy Punyala   
..                                                ...   
84                    Aaditya Sharda, Souvik Sengupta   
85                                     Ritesh Agarwal   
86  Amit Bhasin, Kushal Karwa, Nitin Rana, Rishabh...   
87  Niraj Singh, Ramanshu Mahaur, Ganesh Pawar, Mo...   
88                                     Rajendra Setia   

                                             Investor     Amount($)  \
0                                Sixth Sense Ventures    $6,300,000   
1                                    General Atlantic  $150,000,000   
2      Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3   Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4            Innovation in Food and Agriculture (IFA)    $6,000,000   
..                                                ...           ...   
84  Tiger Global, Nexus Venture Partners, Accel Pa...   $20,000,000   
85  MyPreferred Transformation, Avendus Finance, S...  $693,000,000   
86                                    Sequoia Capital    $5,000,000   
87  Norwest Venture Partners, General Catalyst, Fu...   $50,000,000   
88     TPG, Norwest Venture Partners, Evolvence India   $33,000,000   

            Stage  
0             NaN  
1        Series C  
2   Fresh funding  
3        Series D  
4             NaN  
..            ...  
84       Series A  
85            NaN  
86       Series B  
87            NaN  
88            NaN  

[89 rows x 9 columns]

In [27]:
data19.head()

Company/Brand  Founded HeadQuarter           Sector  \
0  Bombay Shaving      NaN         NaN        Ecommerce   
1       Ruangguru   2014.0      Mumbai           Edtech   
2        Eduisfun      NaN      Mumbai           Edtech   
3        HomeLane   2014.0     Chennai  Interior design   
4        Nu Genes   2004.0   Telangana         AgriTech   

                                        What it does  \
0         Provides a range of male grooming products   
1  A learning platform that provides topic-based ...   
2            It aims to make learning fun via games.   
3              Provides interior designing solutions   
4  It is a seed company engaged in production, pr...   

                                Founders  \
0                     Shantanu Deshpande   
1  Adamas Belva Syah Devara, Iman Usman.   
2                          Jatin Solanki   
3           Srikanth Iyer, Rama Harinath   
4                 Narayana Reddy Punyala   

                                            Investor     Amount($)  \
0                               Sixth Sense Ventures    $6,300,000   
1                                   General Atlantic  $150,000,000   
2     Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3  Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4           Innovation in Food and Agriculture (IFA)    $6,000,000   

           Stage  
0            NaN  
1       Series C  
2  Fresh funding  
3       Series D  
4            NaN

In [28]:
# Creating a column to aid in identifying each data by the year they were added 
data19['Funding_Year'] = 2019

In [29]:
data19.head()

Company/Brand  Founded HeadQuarter           Sector  \
0  Bombay Shaving      NaN         NaN        Ecommerce   
1       Ruangguru   2014.0      Mumbai           Edtech   
2        Eduisfun      NaN      Mumbai           Edtech   
3        HomeLane   2014.0     Chennai  Interior design   
4        Nu Genes   2004.0   Telangana         AgriTech   

                                        What it does  \
0         Provides a range of male grooming products   
1  A learning platform that provides topic-based ...   
2            It aims to make learning fun via games.   
3              Provides interior designing solutions   
4  It is a seed company engaged in production, pr...   

                                Founders  \
0                     Shantanu Deshpande   
1  Adamas Belva Syah Devara, Iman Usman.   
2                          Jatin Solanki   
3           Srikanth Iyer, Rama Harinath   
4                 Narayana Reddy Punyala   

                                            Investor     Amount($)  \
0                               Sixth Sense Ventures    $6,300,000   
1                                   General Atlantic  $150,000,000   
2     Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3  Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4           Innovation in Food and Agriculture (IFA)    $6,000,000   

           Stage  Funding_Year  
0            NaN          2019  
1       Series C          2019  
2  Fresh funding          2019  
3       Series D          2019  
4            NaN          2019

In [30]:
data19.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 89 entries, 0 to 88
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company/Brand  89 non-null     object 
 1   Founded        60 non-null     float64
 2   HeadQuarter    70 non-null     object 
 3   Sector         84 non-null     object 
 4   What it does   89 non-null     object 
 5   Founders       86 non-null     object 
 6   Investor       89 non-null     object 
 7   Amount($)      89 non-null     object 
 8   Stage          43 non-null     object 
 9   Funding_Year   89 non-null     int64  
dtypes: float64(1), int64(1), object(8)
memory usage: 7.1+ KB


In [31]:
data19.shape

(89, 10)

In [32]:
# printing out dataframe column names 
print(data19.columns)

Index(['Company/Brand', 'Founded', 'HeadQuarter', 'Sector', 'What it does',
       'Founders', 'Investor', 'Amount($)', 'Stage', 'Funding_Year'],
      dtype='object')


Loading Dataset into VSO Environment 

3. Data Collection 2018


In [33]:
# Accessing database for 2019 from the Github Repository, now downloaded unto the local pc 
data18 = pd.read_csv('startup_funding2018.csv') 
data18

Company Name                                           Industry  \
0      TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1      Happy Cow Dairy                               Agriculture, Farming   
2           MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3          PayMe India                        Financial Services, FinTech   
4             Eunimart                 E-Commerce Platforms, Retail, SaaS   
..                 ...                                                ...   
521              Udaan   B2B, Business Development, Internet, Marketplace   
522  Happyeasygo Group                                    Tourism, Travel   
523             Mombay         Food and Beverage, Food Delivery, Internet   
524         Droni Tech                             Information Technology   
525            Netmeds         Biotechnology, Health Care, Pharmaceutical   

    Round/Series       Amount                          Location  \
0           Seed       250000       Bangalore, Karnataka, India   
1           Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2       Series A  ₹65,000,000           Gurgaon, Haryana, India   
3          Angel      2000000       Noida, Uttar Pradesh, India   
4           Seed            —  Hyderabad, Andhra Pradesh, India   
..           ...          ...                               ...   
521     Series C    225000000       Bangalore, Karnataka, India   
522     Series A            —           Haryana, Haryana, India   
523         Seed         7500        Mumbai, Maharashtra, India   
524         Seed  ₹35,000,000        Mumbai, Maharashtra, India   
525     Series C     35000000        Chennai, Tamil Nadu, India   

                                         About Company  
0    TheCollegeFever is a hub for fun, fiesta and f...  
1    A startup which aggregates milk from dairy far...  
2            Leading Online Loans Marketplace in India  
3    PayMe India is an innovative FinTech organizat...  
4    Eunimart is a one stop solution for merchants ...  
..                                                 ...  
521  Udaan is a B2B trade platform, designed specif...  
522            HappyEasyGo is an online travel domain.  
523  Mombay is a unique opportunity for housewives ...  
524  Droni Tech manufacture UAVs and develop softwa...  
525       Welcome to India's most convenient pharmacy!  

[526 rows x 6 columns]

In [34]:
data18.head()

Company Name                                           Industry  \
0  TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1  Happy Cow Dairy                               Agriculture, Farming   
2       MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3      PayMe India                        Financial Services, FinTech   
4         Eunimart                 E-Commerce Platforms, Retail, SaaS   

  Round/Series       Amount                          Location  \
0         Seed       250000       Bangalore, Karnataka, India   
1         Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2     Series A  ₹65,000,000           Gurgaon, Haryana, India   
3        Angel      2000000       Noida, Uttar Pradesh, India   
4         Seed            —  Hyderabad, Andhra Pradesh, India   

                                       About Company  
0  TheCollegeFever is a hub for fun, fiesta and f...  
1  A startup which aggregates milk from dairy far...  
2          Leading Online Loans Marketplace in India  
3  PayMe India is an innovative FinTech organizat...  
4  Eunimart is a one stop solution for merchants ...

In [35]:
# Creating a column to aid in identifying each data by the year they were added 
data18['Funding_year'] = 2018
data18

Company Name                                           Industry  \
0      TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1      Happy Cow Dairy                               Agriculture, Farming   
2           MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3          PayMe India                        Financial Services, FinTech   
4             Eunimart                 E-Commerce Platforms, Retail, SaaS   
..                 ...                                                ...   
521              Udaan   B2B, Business Development, Internet, Marketplace   
522  Happyeasygo Group                                    Tourism, Travel   
523             Mombay         Food and Beverage, Food Delivery, Internet   
524         Droni Tech                             Information Technology   
525            Netmeds         Biotechnology, Health Care, Pharmaceutical   

    Round/Series       Amount                          Location  \
0           Seed       250000       Bangalore, Karnataka, India   
1           Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2       Series A  ₹65,000,000           Gurgaon, Haryana, India   
3          Angel      2000000       Noida, Uttar Pradesh, India   
4           Seed            —  Hyderabad, Andhra Pradesh, India   
..           ...          ...                               ...   
521     Series C    225000000       Bangalore, Karnataka, India   
522     Series A            —           Haryana, Haryana, India   
523         Seed         7500        Mumbai, Maharashtra, India   
524         Seed  ₹35,000,000        Mumbai, Maharashtra, India   
525     Series C     35000000        Chennai, Tamil Nadu, India   

                                         About Company  Funding_year  
0    TheCollegeFever is a hub for fun, fiesta and f...          2018  
1    A startup which aggregates milk from dairy far...          2018  
2            Leading Online Loans Marketplace in India          2018  
3    PayMe India is an innovative FinTech organizat...          2018  
4    Eunimart is a one stop solution for merchants ...          2018  
..                                                 ...           ...  
521  Udaan is a B2B trade platform, designed specif...          2018  
522            HappyEasyGo is an online travel domain.          2018  
523  Mombay is a unique opportunity for housewives ...          2018  
524  Droni Tech manufacture UAVs and develop softwa...          2018  
525       Welcome to India's most convenient pharmacy!          2018  

[526 rows x 7 columns]

In [36]:
data18.head()

Company Name                                           Industry  \
0  TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1  Happy Cow Dairy                               Agriculture, Farming   
2       MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3      PayMe India                        Financial Services, FinTech   
4         Eunimart                 E-Commerce Platforms, Retail, SaaS   

  Round/Series       Amount                          Location  \
0         Seed       250000       Bangalore, Karnataka, India   
1         Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2     Series A  ₹65,000,000           Gurgaon, Haryana, India   
3        Angel      2000000       Noida, Uttar Pradesh, India   
4         Seed            —  Hyderabad, Andhra Pradesh, India   

                                       About Company  Funding_year  
0  TheCollegeFever is a hub for fun, fiesta and f...          2018  
1  A startup which aggregates milk from dairy far...          2018  
2          Leading Online Loans Marketplace in India          2018  
3  PayMe India is an innovative FinTech organizat...          2018  
4  Eunimart is a one stop solution for merchants ...          2018

In [37]:
data18.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 526 entries, 0 to 525
Data columns (total 7 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Company Name   526 non-null    object
 1   Industry       526 non-null    object
 2   Round/Series   526 non-null    object
 3   Amount         526 non-null    object
 4   Location       526 non-null    object
 5   About Company  526 non-null    object
 6   Funding_year   526 non-null    int64 
dtypes: int64(1), object(6)
memory usage: 28.9+ KB


In [38]:
data18.shape

(526, 7)

In [39]:
# Renaming some columns 
data20.rename(columns = {'Company_Brand' :'Company_Name'}, inplace =True)

data20.rename(columns = {'HeadQuarter': 'Location'}, inplace =True)

data20.head()

Company_Name  Founded   Location              Sector  \
0    Aqgromalin   2019.0    Chennai            AgriTech   
1      Krayonnz   2019.0  Bangalore              EdTech   
2  PadCare Labs   2018.0       Pune  Hygiene management   
3         NCOME   2020.0  New Delhi              Escrow   
4    Gramophone   2016.0     Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  \
0                      Angel investors  200000.0      None     None   
1                      GSF Accelerator  100000.0  Pre-seed     None   
2                       Venture Center       NaN  Pre-seed     None   
3  Venture Catalysts, PointOne Capital  400000.0      None     None   
4  Siana Capital Management, Info Edge  340000.0      None     None   

   Funding_Year  
0          2020  
1          2020  
2          2020  
3          2020  
4          2020

In [40]:
# Selecting some specific columns from the Dataframe 
data20 = data20[['Company_Name', 'Founded','Location','Sector','Investor','Amount','Stage','Funding_Year']]
# these columns are deemed more important for the very analysis and comparisons I am loking forward to doing             
data20.head() 

Company_Name  Founded   Location              Sector  \
0    Aqgromalin   2019.0    Chennai            AgriTech   
1      Krayonnz   2019.0  Bangalore              EdTech   
2  PadCare Labs   2018.0       Pune  Hygiene management   
3         NCOME   2020.0  New Delhi              Escrow   
4    Gramophone   2016.0     Indore            AgriTech   

                              Investor    Amount     Stage  Funding_Year  
0                      Angel investors  200000.0      None          2020  
1                      GSF Accelerator  100000.0  Pre-seed          2020  
2                       Venture Center       NaN  Pre-seed          2020  
3  Venture Catalysts, PointOne Capital  400000.0      None          2020  
4  Siana Capital Management, Info Edge  340000.0      None          2020

In [41]:
# Converting the funded column to numeric data
data20['Founded'] = pd.to_numeric(data20['Founded'], errors='coerce').convert_dtypes(int)

# EXPLORATORY DATA ANALYSIS 


In [42]:
data18.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 526 entries, 0 to 525
Data columns (total 7 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Company Name   526 non-null    object
 1   Industry       526 non-null    object
 2   Round/Series   526 non-null    object
 3   Amount         526 non-null    object
 4   Location       526 non-null    object
 5   About Company  526 non-null    object
 6   Funding_year   526 non-null    int64 
dtypes: int64(1), object(6)
memory usage: 28.9+ KB


In [43]:
data18.describe().T

count    mean  std     min     25%     50%     75%     max
Funding_year  526.0  2018.0  0.0  2018.0  2018.0  2018.0  2018.0  2018.0

In [44]:
data18.shape

(526, 7)

In [45]:
# Checking for Missing Values in 2018 data 
print(data18.isna().sum())

Company Name     0
Industry         0
Round/Series     0
Amount           0
Location         0
About Company    0
Funding_year     0
dtype: int64


In [46]:
# Checking for duplicated vales in 2018 data 
print(data18.duplicated().sum())

1


In [47]:
data19.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 89 entries, 0 to 88
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company/Brand  89 non-null     object 
 1   Founded        60 non-null     float64
 2   HeadQuarter    70 non-null     object 
 3   Sector         84 non-null     object 
 4   What it does   89 non-null     object 
 5   Founders       86 non-null     object 
 6   Investor       89 non-null     object 
 7   Amount($)      89 non-null     object 
 8   Stage          43 non-null     object 
 9   Funding_Year   89 non-null     int64  
dtypes: float64(1), int64(1), object(8)
memory usage: 7.1+ KB


In [48]:
data19.describe().T

count         mean       std     min     25%     50%      75%  \
Founded        60.0  2014.533333  2.937003  2004.0  2013.0  2015.0  2016.25   
Funding_Year   89.0  2019.000000  0.000000  2019.0  2019.0  2019.0  2019.00   

                 max  
Founded       2019.0  
Funding_Year  2019.0

In [49]:
data19.shape

(89, 10)

In [50]:
# Checking for missing values in 2019 data
print(data19.isna().sum())

Company/Brand     0
Founded          29
HeadQuarter      19
Sector            5
What it does      0
Founders          3
Investor          0
Amount($)         0
Stage            46
Funding_Year      0
dtype: int64


In [51]:
# Checking for duplicated values in 2019 data
print(data19.duplicated().sum())

0


In [52]:
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 8 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   Company_Name  1055 non-null   object 
 1   Founded       842 non-null    Int64  
 2   Location      961 non-null    object 
 3   Sector        1042 non-null   object 
 4   Investor      1017 non-null   object 
 5   Amount        801 non-null    float64
 6   Stage         591 non-null    object 
 7   Funding_Year  1055 non-null   int32  
dtypes: Int64(1), float64(1), int32(1), object(5)
memory usage: 63.0+ KB


In [53]:
data20.shape

(1055, 8)

In [54]:
# Checking for Missing Values in 2020 data 
print(data20.isna().sum())

Company_Name      0
Founded         213
Location         94
Sector           13
Investor         38
Amount          254
Stage           464
Funding_Year      0
dtype: int64


In [55]:
data20.describe().T

count              mean                std      min        25%  \
Founded        842.0        2015.36342           4.097909   1973.0     2014.0   
Amount         801.0  113042969.543071  2476634939.888347  12700.0  1000000.0   
Funding_Year  1055.0            2020.0                0.0   2020.0     2020.0   

                    50%         75%            max  
Founded          2016.0      2018.0         2020.0  
Amount        3000000.0  11000000.0  70000000000.0  
Funding_Year     2020.0      2020.0         2020.0

In [56]:
# Checking for duplicated values in 2020 data 
print(data20.duplicated().sum())

3


In [57]:
data21.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1209 entries, 0 to 1208
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1209 non-null   object 
 1   Founded        1208 non-null   float64
 2   HeadQuarter    1208 non-null   object 
 3   Sector         1209 non-null   object 
 4   What_it_does   1209 non-null   object 
 5   Founders       1205 non-null   object 
 6   Investor       1147 non-null   object 
 7   Amount         1206 non-null   object 
 8   Stage          781 non-null    object 
 9   Funding_Year   1209 non-null   int64  
dtypes: float64(1), int64(1), object(8)
memory usage: 94.6+ KB


In [58]:
data21.shape

(1209, 10)

In [59]:
data21.describe().T

count         mean       std     min     25%     50%     75%  \
Founded       1208.0  2016.655629  4.517364  1963.0  2015.0  2018.0  2020.0   
Funding_Year  1209.0  2021.000000  0.000000  2021.0  2021.0  2021.0  2021.0   

                 max  
Founded       2021.0  
Funding_Year  2021.0

In [60]:
# Checking for missing values in 2021 data 
print(data21.isna().sum())

Company_Brand      0
Founded            1
HeadQuarter        1
Sector             0
What_it_does       0
Founders           4
Investor          62
Amount             3
Stage            428
Funding_Year       0
dtype: int64


In [61]:
data21.duplicated()

0       False
1       False
2       False
3       False
4       False
        ...  
1204    False
1205    False
1206    False
1207    False
1208    False
Length: 1209, dtype: bool

In [62]:
# Checking for Duplicate Values in 2021 data
print(data21.duplicated().sum())

19


# DATA CLEANING 

Year 2021

In the data exploratory stage, the following were identified;
1. A total of 19 duplicates in the dataset.
2. There are missing values in 6 columns; Founded(1), HeadQuarter(1), Founders(4), Investor(62), Amount(3), and Stage(428).
3. The Amount data type shows object instead of int64, or float64.
4. The sector column should be categorized properly.

In [63]:
# Filtering the Duplicated Values in  2021 dataset 
data21[data21.duplicated()]

Company_Brand  Founded             HeadQuarter  \
107           Curefoods   2020.0               Bangalore   
109            Bewakoof   2012.0                  Mumbai   
111             FanPlay   2020.0          Computer Games   
117      Advantage Club   2014.0                  Mumbai   
119              Ruptok   2020.0               New Delhi   
243            Trinkerr   2021.0               Bangalore   
244               Zorro   2021.0                Gurugram   
245       Ultraviolette   2021.0               Bangalore   
246          NephroPlus   2009.0               Hyderabad   
247             Unremot   2020.0               Bangalore   
248         FanAnywhere   2021.0               Bangalore   
249          PingoLearn   2021.0                    Pune   
250                Spry   2021.0                  Mumbai   
251             Enmovil   2015.0               Hyderabad   
252       ASQI Advisors   2019.0                  Mumbai   
253  Insurance Samadhan   2018.0               New Delhi   
254     Evenflow Brands   2020.0                  Mumbai   
255          MasterChow   2020.0        Food & Beverages   
256  Fullife Healthcare   2009.0  Pharmaceuticals\t#REF!   

                                                Sector  \
107                                   Food & Beverages   
109                                  Apparel & Fashion   
111                                     Computer Games   
117                                             HRTech   
119                                            FinTech   
243                                    Capital Markets   
244                                     Social network   
245                                         Automotive   
246                             Hospital & Health Care   
247                  Information Technology & Services   
248                                 Financial Services   
249                                         E-learning   
250                                              Music   
251                  Information Technology & Services   
252                                 Financial Services   
253                                          Insurance   
254                                     Consumer Goods   
255                                          Hauz Khas   
256  Primary Business is Development and Manufactur...   

                                          What_it_does  \
107  Healthy & nutritious foods and cold pressed ju...   
109  Bewakoof is a lifestyle fashion brand that mak...   
111  A real money game app specializing in trivia g...   
117  Advantage Club is India's largest employee eng...   
119  Ruptok fintech Pvt. Ltd. is an online gold loa...   
243  Trinkerr is India's first social trading platf...   
244               Pseudonymous social network platform   
245  Create and Inspire the future of sustainable u...   
246  A vision and passion of redefining healthcare ...   
247      Unremot is a personal office for consultants!   
248                             Celebrity NFT platform   
249  PingoLearn offers language learning courses wi...   
250  THE BEST ROYALTY-FREE MUSIC TO CREATE TRENDING...   
251  Enmovil delivers a Business Intelligence Platf...   
252  Bringing Blockchain technology intro mainstrea...   
253  The insurance industry will help in representa...   
254  Identify and partner with the upcoming eCommer...   
255                A ready-to-cook Asian cuisine brand   
256                                       Varun Khanna   

                                          Founders  \
107                                   Ankit Nagori   
109                               Prabhkiran Singh   
111                                         YC W21   
117             Sourabh Deorah, Smiti Bhatt Deorah   
119                                    Ankur Gupta   
243                Manvendra Singh, Gaurav Agarwal   
244  Jasveer Singh, Abhishek Asthana, Deepak Kumar   
245            Subramaniam Narayan, Niraj Rajmohan   
2

In [64]:
# Dropping Duplicates 
data21.drop_duplicates(inplace = True)

In [65]:
# Confirming the Changes made with regards to duplicates 
data21.duplicated().sum()

0

All duplicate data have been dropped in the 2021 dataset to make it concise. 

In [66]:
# Entries in Worng columns 
data21.loc[[98, 241, 242, 257, 538, 545, 551, 674, 1100, 1148, 1176]]

Company_Brand  Founded                        HeadQuarter  \
98               FanPlay   2020.0                     Computer Games   
241           MasterChow   2020.0                   Food & Beverages   
242   Fullife Healthcare   2009.0             Pharmaceuticals\t#REF!   
257              MoEVing   2021.0                    Gurugram\t#REF!   
538          Little Leap   2020.0                          New Delhi   
545            AdmitKard   2016.0                              Noida   
551                BHyve   2020.0                             Mumbai   
674         MYRE Capital   2020.0                             Mumbai   
1100            Sochcast   2020.0                Online Media\t#REF!   
1148           Godamwale   2016.0                             Mumbai   
1176                Peak   2014.0  Information Technology & Services   

                                                 Sector  \
98                                       Computer Games   
241                                           Hauz Khas   
242   Primary Business is Development and Manufactur...   
257   MoEVing is India's only Electric Mobility focu...   
538                                              EdTech   
545                                              EdTech   
551                                     Human Resources   
674                              Commercial Real Estate   
1100  Sochcast is an Audio experiences company that ...   
1148                           Logistics & Supply Chain   
1176                     Manchester, Greater Manchester   

                                           What_it_does  \
98    A real money game app specializing in trivia g...   
241                 A ready-to-cook Asian cuisine brand   
242                                        Varun Khanna   
257                         Vikash Mishra, Mragank Jain   
538                 Soft Skills that make Smart Leaders   
545   A tech solution for end to end career advisory...   
551   A Future of Work Platform for diffusing Employ...   
674                 Democratising Real Estate Ownership   
1100  CA Harvinderjit Singh Bhatia, Garima Surana, A...   
1148  Godamwale is tech enabled integrated logistics...   
1176  Peak helps the world's smartest companies put ...   

                                               Founders  \
98                                               YC W21   
241                       Vidur Kataria, Sidhanth Madan   
242                  Morgan Stanley Private Equity Asia   
257      Anshuman Maheshwary, Dr Srihari Raju Kalidindi   
538   Holistic Development Programs for children in ...   
545    Vamsi Krishna, Pulkit Jain, Gaurav Munjal\t#REF!   
551                                   Backed by 100x.VC   
674             Own rent yielding commercial properties   
1100            Vinners, Raj Nayak, Amritaanshu Agrawal   
1148          Basant Kumar, Vivek Tiwari, Ranbir Nandan   
1176                                        Atul Sharma   

                             Investor                          Amount  \
98        Pritesh Kumar, Bharat Gupta                        Upsparks   
241                      WEH Ventures                         $461000   
242                         $22000000                        Series C   
257                          $5000000                            Seed   
538                      Vishal Gupta                    ah! Ventures   
545                          $1000000                    Pre-series A   
551   Omkar Pandharkame, Ketaki Ogale  ITO Angel Network, LetsVenture   
674                       Aryaman Vir                            None   
1100                     $Undisclosed                            None   
1148                   1000000\t#REF!                            Seed   
1176           SoftBank Vision Fund 2                       $75000000   

         Stage  Funding_Year  
98    $1200000          2021  
241       Seed          2021  
242       None          2021  
257       None    

In [67]:
# Correcting Entries in Wrong Columns 

# LABEL 98 (FanPlay)
data21.loc[98, "Amount"] = "$1200000"
data21.loc[98, "Stage"] = "Undisclosed"
data21.loc[98, "Investor"] = "Upsparks"
data21.loc[98, "HeadQuarter"] = "undisclosed"

## LABEL 241 (MasterChow)
data21.loc[241,  "HeadQuarter"] = "Hauz Khas"
data21.loc[241, "Sector"] = "Food & Beverages"

#LABEL 257(MoEVing)
data21.loc[257, "Amount"] = "$5000000"
data21.loc[257, "HeadQuarter"] = "Gurugram"
data21.loc[257, "Sector"] = "Electronic Vehicle"
data21.loc[257, "Stage"] = "Seed"
data21.loc[257, "Investor"] = "Anshuman Maheshwary, Dr Srihari Raju Kalidindi"

#LABEL 242, 256(Fullife Healthcare)
data21.loc[242, "Amount"] = "$22000000"
data21.loc[242, "Investor"] = "Morgan Stanley Private Equity Asia"
data21.loc[242, "Stage"] = "Series C"
data21.loc[242, "HeadQuarter"] = "Mumbai"
data21.loc[242, "Sector"] = "Pharmaceuticals"
data21.loc[242, "Founders"] = "Varun Khanna"
data21.loc[242, "What_it_does"] = "Primary Business is Development and Manufacture of sports nutrition and supplements"

#LABEL 538 (Little Leap)
data21.loc[538, "Amount"] = "$300000"
data21.loc[538, "Investor"] = "ah! Ventures"
data21.loc[538, "Stage"] = "Undisclosed"
data21.loc[538, "Founders"] = "Vishal Gupta"

#LABEL 545 (AdmitKard)
data21.loc[545, "Amount"] = "$1000000"
data21.loc[545, "Investor"] = "Unknown"
data21.loc[545, "Stage"] = "Pre-series A"
data21.loc[545, "Founders"] = "Vamsi Krishna, Pulkit Jain, Gaurav Munjal"

#LABEL 551 (BHyve)
data21.loc[551, "Amount"] = "$300000"
data21.loc[551, "Investor"] = "ITO Angel Network, LetsVenture, 100x.VC"
data21.loc[551, "Stage"] = "Undisclosed"
data21.loc[551, "Sector"] = "HR"

#LABEL 674 (MYRE Capital)
data21.loc[674, "Amount"] = "$6000000"
data21.loc[674, "Stage"] = "Undisclosed"

#LABEL 677 (Saarthi Pedagogy)
data21.loc[677, "Amount"] = "$1000000"
data21.loc[677, "Investor"] = "JITO Angel Network, LetsVenture"
data21.loc[677, "Stage"] = "Undisclosed"

#LABEL 1100 (Sochcast)
data21.loc[1100, "Investor"] = "Undisclosed"
data21.loc[1100, "HeadQuarter"] = "Undisclosed"
data21.loc[1100, "Sector"] = "Online Media"

#LABEL 1148 (Godamwale)
data21.loc[1148, "Amount"] = "1000000"
data21.loc[1148, "Investor"] = "Undisclosed"
data21.loc[1148, "Stage"] = "Seed"

#LABEL 1176 (Peak)
data21.loc[1176, "Sector"] = "Information Technology & Services"
data21.loc[1176, "HeadQuarter"] = "Manchester"

data21

Company_Brand  Founded HeadQuarter                 Sector  \
0     Unbox Robotics   2019.0   Bangalore             AI startup   
1             upGrad   2015.0      Mumbai                 EdTech   
2        Lead School   2012.0      Mumbai                 EdTech   
3            Bizongo   2015.0      Mumbai         B2B E-commerce   
4           FypMoney   2021.0    Gurugram                FinTech   
...              ...      ...         ...                    ...   
1204        Gigforce   2019.0    Gurugram  Staffing & Recruiting   
1205          Vahdam   2015.0   New Delhi       Food & Beverages   
1206    Leap Finance   2019.0   Bangalore     Financial Services   
1207    CollegeDekho   2015.0    Gurugram                 EdTech   
1208          WeRize   2019.0   Bangalore     Financial Services   

                                           What_it_does  \
0     Unbox Robotics builds on-demand AI-driven ware...   
1        UpGrad is an online higher education platform.   
2     LEAD School offers technology based school tra...   
3     Bizongo is a business-to-business online marke...   
4     FypMoney is Digital NEO Bank for Teenagers, em...   
...                                                 ...   
1204                  A gig/on-demand staffing company.   
1205  VAHDAM is among the world’s first vertically i...   
1206  International education loans for high potenti...   
1207  Collegedekho.com is Student’s Partner, Friend ...   
1208  India’s first socially distributed full stack ...   

                                               Founders  \
0                           Pramod Ghadge, Shahid Memon   
1     Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                            Smita Deorah, Sumeet Mehta   
3               Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                         Kapil Banwari   
...                                                 ...   
1204                        Chirag Mittal, Anirudh Syal   
1205                                         Bala Sarda   
1206                         Arnav Kumar, Vaibhav Singh   
1207                                       Ruchir Arora   
1208                      Vishal Chopra, Himanshu Gupta   

                                           Investor        Amount  \
0                       BEENEXT, Entrepreneur First    $1,200,000   
1          Unilazer Ventures, IIFL Asset Management  $120,000,000   
2                  GSV Ventures, Westbridge Capital   $30,000,000   
3                            CDC Group, IDG Capital   $51,000,000   
4     Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000   
...                                             ...           ...   
1204                                Endiya Partners      $3000000   
1205                                       IIFL AMC     $20000000   
1206                                   Owl Ventures     $55000000   
1207               Winter Capital, ETS, Man Capital     $26000000   
1208                 3one4 Capital, Kalaari Capital      $8000000   

             Stage  Funding_Year  
0     Pre-series A          2021  
1             None          2021  
2         Series D          2021  
3         Series C          2021  
4             Seed          2021  
...            ...           ...  
1204  Pre-series A          2021  
1205      Series D          2021  
1206      Series C          2021  
1207      Series B          2021  
1208      Series A          2021  

[1190 rows x 10 columns]

For analysis of dataset location, cross-referencing ad manual inspections were the tools employed as they were few in number

In [68]:
# Cleaning the column names of the 2021 dataset
data21 = data21.rename(columns=lambda x: x.lower().replace(' ', '_'))
# Rename company_brand as company/brand
data21 = data21.rename(columns={"company_brand":"company/brand"})
# Adding the year column
data21["funded_year"] = 2021
data21

company/brand  founded headquarter                 sector  \
0     Unbox Robotics   2019.0   Bangalore             AI startup   
1             upGrad   2015.0      Mumbai                 EdTech   
2        Lead School   2012.0      Mumbai                 EdTech   
3            Bizongo   2015.0      Mumbai         B2B E-commerce   
4           FypMoney   2021.0    Gurugram                FinTech   
...              ...      ...         ...                    ...   
1204        Gigforce   2019.0    Gurugram  Staffing & Recruiting   
1205          Vahdam   2015.0   New Delhi       Food & Beverages   
1206    Leap Finance   2019.0   Bangalore     Financial Services   
1207    CollegeDekho   2015.0    Gurugram                 EdTech   
1208          WeRize   2019.0   Bangalore     Financial Services   

                                           what_it_does  \
0     Unbox Robotics builds on-demand AI-driven ware...   
1        UpGrad is an online higher education platform.   
2     LEAD School offers technology based school tra...   
3     Bizongo is a business-to-business online marke...   
4     FypMoney is Digital NEO Bank for Teenagers, em...   
...                                                 ...   
1204                  A gig/on-demand staffing company.   
1205  VAHDAM is among the world’s first vertically i...   
1206  International education loans for high potenti...   
1207  Collegedekho.com is Student’s Partner, Friend ...   
1208  India’s first socially distributed full stack ...   

                                               founders  \
0                           Pramod Ghadge, Shahid Memon   
1     Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                            Smita Deorah, Sumeet Mehta   
3               Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                         Kapil Banwari   
...                                                 ...   
1204                        Chirag Mittal, Anirudh Syal   
1205                                         Bala Sarda   
1206                         Arnav Kumar, Vaibhav Singh   
1207                                       Ruchir Arora   
1208                      Vishal Chopra, Himanshu Gupta   

                                           investor        amount  \
0                       BEENEXT, Entrepreneur First    $1,200,000   
1          Unilazer Ventures, IIFL Asset Management  $120,000,000   
2                  GSV Ventures, Westbridge Capital   $30,000,000   
3                            CDC Group, IDG Capital   $51,000,000   
4     Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000   
...                                             ...           ...   
1204                                Endiya Partners      $3000000   
1205                                       IIFL AMC     $20000000   
1206                                   Owl Ventures     $55000000   
1207               Winter Capital, ETS, Man Capital     $26000000   
1208                 3one4 Capital, Kalaari Capital      $8000000   

             stage  funding_year  funded_year  
0     Pre-series A          2021         2021  
1             None          2021         2021  
2         Series D          2021         2021  
3         Series C          2021         2021  
4             Seed          2021         2021  
...            ...           ...          ...  
1204  Pre-series A          2021         2021  
1205      Series D          2021         2021  
1206      Series C          2021         2021  
1207      Series B          2021         2021  
1208      Series A          2021         2021  

[1190 rows x 11 columns]

Exchange rates 

Source: OFX

exchange_rates = {
    2018: 0.014649,
    2019: 0.014209,
    2020: 0.013501,
    2021: 0.013527
}

In [69]:
# Cleaning the Amount Column 
# Checking for values that do not have the dollar sign attached to them 
data21[~data21.amount.str.get(0).isin(['$'])]

company/brand  founded  headquarter                    sector  \
7                 Qube Health   2016.0       Mumbai                HealthTech   
8                    Vitra.ai   2020.0    Bangalore              Tech Startup   
21                      Uable   2020.0    Bangalore                    EdTech   
39                   TruNativ   2019.0       Mumbai          Food & Beverages   
54                     AntWak   2019.0    Bangalore                    EdTech   
64                Rage Coffee   2018.0    New Delhi          Food & Beverages   
67                      Kudos   2014.0         Pune                   FinTech   
86                   Onsurity   2020.0    Bangalore                HealthCare   
316                 Hubhopper   2015.0    New Delhi                   Podcast   
319             Battery Smart   2019.0    New Delhi                   Battery   
321                   Onelife   2019.0       Mumbai                Healthcare   
325                     Aisle   2014.0    Bangalore                    Dating   
334                   Zingbus   2019.0      Haryana            Transportation   
336                  Codingal   2020.0    Bangalore                    EdTech   
345                Hypd Store   2020.0    New Delhi          Content commerce   
367                 PrepBytes   2019.0     Gurugram                    EdTech   
376                     ExMyB   2018.0     Gurugram                Consulting   
379                      Qin1   2019.0        Noida                    EdTech   
383                   KIKO TV   2020.0       Mumbai                E-commerce   
384               Quali55Care   2018.0       Mumbai               Health care   
386                    RevFin   2018.0    New Delhi                   FinTech   
393                      Zouk   2016.0       Mumbai     Fashion and lifestyle   
406           Digit Insurance   2016.0    Bangalore                   FinTech   
429                    Bolkar   2018.0    Bangalore            Social network   
430           Wright Research   2019.0       Mumbai             Advisory firm   
437            Oneiric Gaming   2020.0   Chandigarh                    Gaming   
452                   Agri10x   2018.0         Pune                  AgriTech   
724                 Instoried   2018.0    Bangalore          Content creation   
732                  Homingos   2018.0    Bangalore         Augmented reality   
733   Anar Business Community   2020.0       Mumbai              Tech Startup   
735                   Settlrs   2018.0    Bangalore              Rental space   
766                      Wysa   2015.0    Bangalore                AI startup   
769                   Firmway   2016.0       Mumbai                   FinTech   
794                    Bijnis   2014.0    New Delhi               B2B service   
801                   Virohan   2018.0     Gurugram                    EdTech   
807                 ParkSmart   2018.0    New Delhi                Automotive   
810              Spark Studio   2020.0    Bangalore                    EdTech   
814                     Keito   2016.0         Pune                AI company   
820                  LegalPay   2019.0    New Delhi                   FinTech   
824               Avalon Labs   2017.0    Bangalore                   FinTech   
827                   Rezo.ai   2017.0        Noida                AI startup   
833                   Polygon   2017.0       Mumbai                    Crypto   
846                  Ingenium   2018.0    New Delhi                    EdTech   
853                   Celcius   2020.0       Mumbai                 Logistics   
1100                 Sochcast   2020.0  Undisclosed              Online Media   
1148                Godamwale   2016.0       Mumbai  Logistics & Supply Chain   

                                           what_it_does  \
7     India's Most Respected Workplace Healthcare Ma...   
8     Vitra.ai is an AI-based video translation plat...   
21    Uable offers role based p

In [70]:
data21.amount.unique()

array(['$1,200,000', '$120,000,000', '$30,000,000', '$51,000,000',
       '$2,000,000', '$188,000,000', '$200,000', 'Undisclosed',
       '$1,000,000', '$3,000,000', '$100,000', '$700,000', '$9,000,000',
       '$40,000,000', '$49,000,000', '$400,000', '$300,000',
       '$25,000,000', '$160,000,000', '$150,000', '$1,800,000',
       '$5,000,000', '$850,000', '$53,000,000', '$500,000', '$1,100,000',
       '$6,000,000', '$800,000', '$10,000,000', '$21,000,000',
       '$7,500,000', '$26,000,000', '$7,400,000', '$1,500,000',
       '$600,000', '$800,000,000', '$17,000,000', '$3,500,000',
       '$15,000,000', '$215,000,000', '$2,500,000', '$350,000,000',
       '$5,500,000', '$83,000,000', '$110,000,000', '$500,000,000',
       '$65,000,000', '$150,000,000,000', '$300,000,000', '$2,200,000',
       '$35,000,000', '$140,000,000', '$4,000,000', '$13,000,000', None,
       '$Undisclosed', '$2000000', '$800000', '$6000000', '$2500000',
       '$9500000', '$13000000', '$5000000', '$8000000',

In [71]:
# Remove the word with a value
data21['amount'] = data21['amount'].str.replace("1000000\t#REF!" , '1000000').replace('$Undisclosed', np.NaN).replace('$undisclosed',np.NaN).replace('Undisclosed', np.NaN)


In [72]:
data21.amount.unique()

array(['$1,200,000', '$120,000,000', '$30,000,000', '$51,000,000',
       '$2,000,000', '$188,000,000', '$200,000', nan, '$1,000,000',
       '$3,000,000', '$100,000', '$700,000', '$9,000,000', '$40,000,000',
       '$49,000,000', '$400,000', '$300,000', '$25,000,000',
       '$160,000,000', '$150,000', '$1,800,000', '$5,000,000', '$850,000',
       '$53,000,000', '$500,000', '$1,100,000', '$6,000,000', '$800,000',
       '$10,000,000', '$21,000,000', '$7,500,000', '$26,000,000',
       '$7,400,000', '$1,500,000', '$600,000', '$800,000,000',
       '$17,000,000', '$3,500,000', '$15,000,000', '$215,000,000',
       '$2,500,000', '$350,000,000', '$5,500,000', '$83,000,000',
       '$110,000,000', '$500,000,000', '$65,000,000', '$150,000,000,000',
       '$300,000,000', '$2,200,000', '$35,000,000', '$140,000,000',
       '$4,000,000', '$13,000,000', None, '$2000000', '$800000',
       '$6000000', '$2500000', '$9500000', '$13000000', '$5000000',
       '$8000000', '$1000000', '$1200000', '

In [73]:
# Creating a function on amount column for data21
def clean_amount():
    # copy original amount columns
    amount_column = data21["amount"].copy().str.replace(",","") 
    
    # extract values in dollars($)
    amount_in_dollar_1 = amount_column[amount_column.str.startswith("$", na=False)]
    # strip off the dollar symbol
    amount_in_dollar_1 = amount_in_dollar_1.str.lstrip("$")

    # extract values in dollars($)
    amount_in_dollar_2 = amount_column[amount_column.str.startswith("$$", na=False)]

     # strip off the dollar symbol
    amount_in_dollar_2 = amount_in_dollar_2.str.lstrip("$$")
    # Replace the Unclean columns with the clean one

    amount_column.loc[amount_in_dollar_1.index] = amount_in_dollar_1
    amount_column.loc[amount_in_dollar_2.index] = amount_in_dollar_2
    
    # convert the clean column to numeric
    amount_column = pd.to_numeric(amount_column,errors="coerce")
    # update the amount column 
    data21["amount"] = amount_column
    
    data21

In [74]:
# Checking for null values in the amount column
data21["amount"].isna().sum()
data21["amount"]

0         $1,200,000
1       $120,000,000
2        $30,000,000
3        $51,000,000
4         $2,000,000
            ...     
1204        $3000000
1205       $20000000
1206       $55000000
1207       $26000000
1208        $8000000
Name: amount, Length: 1190, dtype: object

In [75]:
# Cleaning the stage column 
modified_stage= data21.stage.unique()
modified_stage

array(['Pre-series A', None, 'Series D', 'Series C', 'Seed', 'Series B',
       'Series E', 'Pre-seed', 'Series A', 'Pre-series B', 'Debt',
       'Undisclosed', 'Bridge', 'Seed+', 'Series F2', 'Series A+',
       'Series G', 'Series F', 'Series H', 'Series B3', 'PE', 'Series F1',
       'Pre-series A1', 'Early seed', 'Series D1', 'Seies A',
       'Pre-series', 'Series A2', 'Series I'], dtype=object)

In [76]:
data21["amount"].isna().sum()

139

In [77]:
# Performing Descriptive Statistics after light cleaning of the amount column 
data21.describe(include="all").T

count unique  \
company/brand    1190   1033   
founded        1189.0    NaN   
headquarter      1189     68   
sector           1190    252   
what_it_does     1190   1143   
founders         1186   1095   
investor         1129    933   
amount           1051    269   
stage             773     28   
funding_year   1190.0    NaN   
funded_year    1190.0    NaN   

                                                             top freq  \
company/brand                                           BharatPe    8   
founded                                                      NaN  NaN   
headquarter                                            Bangalore  421   
sector                                                   FinTech  121   
what_it_does   BharatPe develops a QR code-based payment app ...    4   
founders                        Ashneer Grover, Shashvat Nakrani    7   
investor                               Inflection Point Ventures   23   
amount                                                  $1000000   52   
stage                                                       Seed  243   
funding_year                                                 NaN  NaN   
funded_year                                                  NaN  NaN   

                      mean       std     min     25%     50%     75%     max  
company/brand          NaN       NaN     NaN     NaN     NaN     NaN     NaN  
founded        2016.634146  4.522098  1963.0  2015.0  2018.0  2020.0  2021.0  
headquarter            NaN       NaN     NaN     NaN     NaN     NaN     NaN  
sector                 NaN       NaN     NaN     NaN     NaN     NaN     NaN  
what_it_does           NaN       NaN     NaN     NaN     NaN     NaN     NaN  
founders               NaN       NaN     NaN     NaN     NaN     NaN     NaN  
investor               NaN       NaN     NaN     NaN     NaN     NaN     NaN  
amount                 NaN       NaN     NaN     NaN     NaN     NaN     NaN  
stage                  NaN       NaN     NaN     NaN     NaN     NaN     NaN  
funding_year        2021.0       0.0  2021.0  2021.0  2021.0  2021.0  2021.0  
funded_year         2021.0       0.0  2021.0  2021.0  2021.0  2021.0  2021.0

In [ ]:
# Creating a function to clean the Amount Column of the 2021 dataset

def clean_amount_2021(Amount):
    try:
        Amount = str(Amount)
        # Remove commas
        Amount = Amount.replace(",", "")
        Amount = Amount.replace('—', "")
        # Check if the value is in Indian Rupees and convert US Dollars: Using USD = 68.4113 which was the average annual Indian Rupee to US Dollars in 2018
        if "₹" in Amount:
            Amount = Amount.replace("₹", "")
            return round(float(Amount) * 0.0146, 2)
        # Check if the value is in US Dollars
        elif "$" in Amount:
            Amount = Amount.replace("$", "")
            return round (float(Amount), 2)
        # check if no currency symbol is present, assume US Dollars
        else:
            return round(float(Amount), 2)
    except ValueError:
        # If the value is not a number, return NaN
        return np.nan
        
# Clean the Amount column of the 2018 DataFrame
data21["Amount"] = data21["Amount"].apply(clean_amount_2021)

KeyError: 'Amount'

Year 2020

In the data exploratory stage, the following were identified;
1. 
2. 
3. 
4. 

In [ ]:
# Coverting the dataset into dataframe 
data20

Company_Name  Founded   Location              Sector  \
0       Aqgromalin     2019    Chennai            AgriTech   
1         Krayonnz     2019  Bangalore              EdTech   
2     PadCare Labs     2018       Pune  Hygiene management   
3            NCOME     2020  New Delhi              Escrow   
4       Gramophone     2016     Indore            AgriTech   
...            ...      ...        ...                 ...   
1050  Leverage Edu     <NA>      Delhi              Edtech   
1051         EpiFi     <NA>       None             Fintech   
1052       Purplle     2012     Mumbai           Cosmetics   
1053        Shuttl     2015      Delhi           Transport   
1054         Pando     2017    Chennai            Logitech   

                                   Investor      Amount       Stage  \
0                           Angel investors    200000.0        None   
1                           GSF Accelerator    100000.0    Pre-seed   
2                            Venture Center         NaN    Pre-seed   
3       Venture Catalysts, PointOne Capital    400000.0        None   
4       Siana Capital Management, Info Edge    340000.0        None   
...                                     ...         ...         ...   
1050  DSG Consumer Partners, Blume Ventures   1500000.0        None   
1051          Sequoia India, Ribbit Capital  13200000.0  Seed Round   
1052                             Verlinvest   8000000.0        None   
1053             SIG Global India Fund LLP.   8043000.0    Series C   
1054                      Chiratae Ventures   9000000.0    Series A   

      Funding_Year  
0             2020  
1             2020  
2             2020  
3             2020  
4             2020  
...            ...  
1050          2020  
1051          2020  
1052          2020  
1053          2020  
1054          2020  

[1055 rows x 8 columns]

In [ ]:
# Removing duplicates from data set
data20.drop_duplicates() 

Company_Name  Founded   Location              Sector  \
0       Aqgromalin     2019    Chennai            AgriTech   
1         Krayonnz     2019  Bangalore              EdTech   
2     PadCare Labs     2018       Pune  Hygiene management   
3            NCOME     2020  New Delhi              Escrow   
4       Gramophone     2016     Indore            AgriTech   
...            ...      ...        ...                 ...   
1050  Leverage Edu     <NA>      Delhi              Edtech   
1051         EpiFi     <NA>       None             Fintech   
1052       Purplle     2012     Mumbai           Cosmetics   
1053        Shuttl     2015      Delhi           Transport   
1054         Pando     2017    Chennai            Logitech   

                                   Investor      Amount       Stage  \
0                           Angel investors    200000.0        None   
1                           GSF Accelerator    100000.0    Pre-seed   
2                            Venture Center         NaN    Pre-seed   
3       Venture Catalysts, PointOne Capital    400000.0        None   
4       Siana Capital Management, Info Edge    340000.0        None   
...                                     ...         ...         ...   
1050  DSG Consumer Partners, Blume Ventures   1500000.0        None   
1051          Sequoia India, Ribbit Capital  13200000.0  Seed Round   
1052                             Verlinvest   8000000.0        None   
1053             SIG Global India Fund LLP.   8043000.0    Series C   
1054                      Chiratae Ventures   9000000.0    Series A   

      Funding_Year  
0             2020  
1             2020  
2             2020  
3             2020  
4             2020  
...            ...  
1050          2020  
1051          2020  
1052          2020  
1053          2020  
1054          2020  

[1052 rows x 8 columns]

In [ ]:
# Confirming changes 
data20

Company_Name  Founded   Location              Sector  \
0       Aqgromalin     2019    Chennai            AgriTech   
1         Krayonnz     2019  Bangalore              EdTech   
2     PadCare Labs     2018       Pune  Hygiene management   
3            NCOME     2020  New Delhi              Escrow   
4       Gramophone     2016     Indore            AgriTech   
...            ...      ...        ...                 ...   
1050  Leverage Edu     <NA>      Delhi              Edtech   
1051         EpiFi     <NA>       None             Fintech   
1052       Purplle     2012     Mumbai           Cosmetics   
1053        Shuttl     2015      Delhi           Transport   
1054         Pando     2017    Chennai            Logitech   

                                   Investor      Amount       Stage  \
0                           Angel investors    200000.0        None   
1                           GSF Accelerator    100000.0    Pre-seed   
2                            Venture Center         NaN    Pre-seed   
3       Venture Catalysts, PointOne Capital    400000.0        None   
4       Siana Capital Management, Info Edge    340000.0        None   
...                                     ...         ...         ...   
1050  DSG Consumer Partners, Blume Ventures   1500000.0        None   
1051          Sequoia India, Ribbit Capital  13200000.0  Seed Round   
1052                             Verlinvest   8000000.0        None   
1053             SIG Global India Fund LLP.   8043000.0    Series C   
1054                      Chiratae Ventures   9000000.0    Series A   

      Funding_Year  
0             2020  
1             2020  
2             2020  
3             2020  
4             2020  
...            ...  
1050          2020  
1051          2020  
1052          2020  
1053          2020  
1054          2020  

[1055 rows x 8 columns]

In [ ]:
# Dropping column 10 because there are a lot of missing values in it and its an unneccessary column
data20.drop(columns='column10', axis=1,inplace=True)


In [ ]:
# Converting the founded column to numeric data

data20['Founded']=pd.to_datetime(data20['Founded'],format='%Y',errors='coerce')
